nerfstudio/fields/nerfacto_field.py

```diff
@@ -112,8 +113,7 @@ def __init__(
         self.use_pred_normals = use_pred_normals
         self.pass_semantic_gradients = pass_semantic_gradients
 
-        base_res: int = 16
-        features_per_level: int = 2
+        base_res: int = 32
```

Comment:
Neuralangelo uses 32 as base_res, maybe we could make it a parameter, but I think bumping to 32 is fairly safe

Reply:
This will probably break existing checkpoints. Maybe make it a parameter with the default being the old value

Reply:
hmm ok